packages/calypso-products/src/translations.tsx

```diff
@@ -1365,9 +1366,9 @@ export const getJetpackProductsWhatIsIncluded = (): Record< string, Array< Trans
 			args: { socialNetworksList },
 			comment: 'Comma separated list of social networks like "Facebook, Mastodon and Tumblr".',
 		} ),
-		translate( 'Engagement Optimizer' ),
+		translate( 'Attach custom images or videos' ),
```

Comment:
Should we be more explicit and say, "Upload custom images or videos with your shares" or something?

Reply:
That's great. What about this?

```diff
+ Upload custom images or videos with your posts
- Upload custom images or videos with your shares
```

Reply:
Sounds good to me!

Reply:
Done. Thanks